lib/diagnostic.js

```diff
@@ -7,22 +7,36 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function () {           //the object min is a function with the perameters ()
+  let min = [];                     // min is an array
+  for (i = 0; i < min.length; i++) {    //loop through the array
+    if (value[i] < min[i]){                //if the value of i is less than min??  that
+                                        //  seems wrong
+    }
+  }
+};
+return currentMin;                  //then return the current min
 };
 
+
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
+const sum = function () {
+  let sum = [];
+  for (i = 0; i < sum.length; i++) {  //iterate through the loop of array sum
+    total += amount[i]; // add each element in sum array to total
 
+  }
+  return total;               //return total
```

Comment:
On line 26, you declared a `const` called sum. On line 27, you declare a `let` also named sum. How is the computer supposed to know which one you want when you call `sum`? If I recall correctly, you've been making the same error in more than 1 diagnostic. Please go back to the fundamentals and review JavaScript variables and functions. Also, try testing your code in node. You will get errors that will tell you what the problem is.